picard/ui/itemviews.py

```diff
@@ -359,7 +362,7 @@ def show_column(self, column, show):
             if self.sectionSize(column) == 0:
                 self.resizeSection(column, self.defaultSectionSize())
             self._visible_columns.add(column)
-            if column == MainPanel.FINGERPRINT_COLUMN:
+            if column in [MainPanel.FINGERPRINT_COLUMN, MainPanel.ACOUSTICBRAINZ_COLUMN]:
```

Comment:
Please use set instead of list, it doesn't make much difference in practice, but `in` set is faster than `in` list:

```suggestion
            if column in {MainPanel.FINGERPRINT_COLUMN, MainPanel.ACOUSTICBRAINZ_COLUMN}:
```